src/options.rs

```diff
@@ -49,6 +51,16 @@ fn parse_range_option(input: &str) -> IResult<&str, UsingOption> {
         |t: (&str, &str)| UsingOption::Range(t.1.to_string()))(input)
 }
 
+fn parse_colnames_option(input: &str) -> IResult<&str, UsingOption> {
+    let option = tag_no_case("COLNAMES");
+
+    let value = preceded(
+        tag("'"), terminated(digit0, tag("'")));
```

Comment:
digit0 should be digit1 probably. Otherwise empty string is allowed as input for this param.